mail-storage/src/main/java/mail/storage/exception/MessageException.java

```diff
@@ -0,0 +1,7 @@
+package mail.storage.exception;
+
+import lombok.experimental.StandardException;
+
+@StandardException
```

Comment:
I would handle it in contoller advuce and this annotation is experimental

Reply:
Fixed.